app/assets/stylesheets/_pickup-history.scss

```diff
@@ -15,7 +15,7 @@
     background-color: $red;
   }
 
-  &.no-donation {
+  &.no_donation {
```

Comment:
@corwinharrell in this case, since the value is coming from Ruby-land where snake_case is favored, is this worth making an exception?